### DISCUSSION TITLE
Configuration with GitVersion.MSBuild

### DISCUSSION THREAD

**Discussion by Xeri7** [GitTools/GitVersion#3875] [Q&A]
Where does the MSBuild version of this pull its configuration from? Presumably it has one, but I have not generated one, so I am not sure where its coming from.

**Comment by asbjornu** [comment]
The configuration file is the same regardless of how GitVersion is executed. By default, it's a `GitVersion.yml` file located in the root of the repository. If you don't have a `GitVersion.yml` file, [GitVersion has a default configuration](https://gitversion.net/docs/reference/configuration) it will use.
